frontend/src/components/crawl-logs.ts

```diff
@@ -131,7 +130,7 @@ export class CrawlLogs extends LitElement {
                     hour="2-digit"
                     minute="2-digit"
                     second="2-digit"
-                    hour-format="24"
+                    hourFormat="24"
```

Comment:
Shoelace docs suggest using the attribute `hour-format` here: https://shoelace.style/components/format-date#hour-formatting